tests/unittest/test_utils.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import numpy as np
+import mxnet as mx
+import gluonnlp as nlp
+
+
+def test_parallel():
```

Comment:
Please mark the test following https://github.com/dmlc/gluon-nlp/pull/417

Reply:
Marked